tools/netlist/netlist.cpp

```diff
@@ -178,6 +187,11 @@ int main(int argc, char** argv) {
         return 1;
     }
 
+    // We don't allow this because in comb loop detection mode we break all netlist loops with edges
+    // so any effort to save/analyze said netlist is futile and dangerous
+    // if (combLoops == true && (astJsonFile || netlistDotFile || fromPointName || toPointName))
+    //    SLANG_THROW(std::runtime_error("Cannot use --comb_loops with other modes"));
+
```

Comment:
Is this dead code?

Reply:
Yes. The original plan was to modify the netlist creation in a way making it not compatible with other commands at the same time. Eventually this was not needed. I'll just remove this.

Reply:
Done